### DISCUSSION TITLE
FreeBSD?

### DISCUSSION THREAD

**Discussion by luckman212** [narrowlink/narrowlink#2] [Q&A] [answered]
Just found out about Narrowlink today via HN. Looks very cool! I am wondering if it can be compiled for FreeBSD 14? Anyone tried yet? (I'm interested in running this on pfSense).

**Comment by SajjadPourali** [comment]
Thanks, I am glad to hear that.

-------
Technically it should be.
You can build it with this command: `cross build --target x86_64-unknown-freebsd --release`
Or alternatively use these binaries: [Nightly Build](https://github.com/narrowlink/narrowlink/releases/tag/Nightly)

Could you please confirm if it works? I can add its build to the nightly builds.

**Comment by luckman212** [comment, marked as answer]
@SajjadPourali Thank you. Well, so far so good—the binaries do seem to run properly on FreeBSD14. I will do some more testing once I have a proper narrowlink config set up.

**Reply by SajjadPourali** [reply]
I have [added](https://github.com/narrowlink/narrowlink/pull/6) x86_64-unknown-freebsd to the _Nightly Build_. Please use the [nightly build](https://github.com/narrowlink/narrowlink/releases/tag/Nightly) binaries.
